modules/nf-core/tabix/bgzip/environment.yml

```diff
@@ -4,4 +4,4 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - bioconda::tabix=1.11
+  - bioconda::htslib=1.19.1
```

Comment:
I'd rather we use the tabix recipe and pin htslib similarly to what is done for the samtools modules, as this is still a tabix modules